src/wrappers/PhyxWrapper.js

```diff
@@ -270,12 +270,19 @@ class PhyxWrapper {
 
     // Finally, add the base URI as an ontology.
     if (baseIRI) jsonld['@id'] = baseIRI;
-    jsonld['@type'] = [owlterms.PHYLOREFERENCE_TEST_CASE, 'owl:Ontology'];
-    jsonld['owl:imports'] = [
-      'http://raw.githubusercontent.com/phyloref/curation-workflow/develop/ontologies/phyloref_testcase.owl',
-      'http://ontology.phyloref.org/2018-12-14/phyloref.owl',
-      'http://ontology.phyloref.org/2018-12-14/tcan.owl',
-    ];
+
+    // Set up the top-level object '@type'. If one is present, we add our terms to that.
+    if (!has(jsonld, '@type')) jsonld['@type'] = [];
+    if (!Array.isArray(jsonld['@type'])) jsonld['@type'] = [jsonld['@type']];
+    jsonld['@type'].push(owlterms.PHYLOREFERENCE_TEST_CASE);
+    jsonld['@type'].push('owl:Ontology');
+
+    // Set up the ontology imports. If one is present, we add our imports to that.
+    if (!has(jsonld, 'owl:imports')) jsonld['owl:imports'] = [];
+    if (!Array.isArray(jsonld['owl:imports'])) jsonld['owl:imports'] = [jsonld['owl:imports']];
+    jsonld['owl:imports'].push('http://raw.githubusercontent.com/phyloref/curation-workflow/develop/ontologies/phyloref_testcase.owl');
+    jsonld['owl:imports'].push('http://ontology.phyloref.org/2018-12-14/phyloref.owl');
```

Comment:
Any reason we have to have the versioned imports rather than just the most current version?

Reply:
The idea here is that this version of Phyx is designed to work with the 2018-12-14 version of the Phyloref.owl and TCAN.owl ontologies. If we make changes to these ontologies, I assume that we will need to make changes in Phyx.js to reflect this change (for instance, if some minor part of the behavior of `phyloref:excludes_lineage_to` changes). That might not happen very often, if at all, but I think it's a good idea to test (and possibly modify) Phyx.js releases when new versions of these ontologies are released.

Part of the problem here is that we can't version these ontologies semantically, so we can't distinguish between breaking changes and non-breaking changes -- I don't think there's any way we can say that generated phyloreferences should use "the latest Phyloref ontology with minor revisions to version 2018-12-14, but not major revisions". In the absence of that, I don't like the idea that phyloreferences previously generated by Phyx.js might start resolving in different ways if the ontologies change, which is why I like versioned imports. But I'm happy to be overruled here.

Reply:
I see and agree with your point.